CONTRIBUTING.md

```diff
@@ -212,6 +212,7 @@ You can access the frontend at [localhost:3000](http://localhost:3000). Whatever
     ```
     docker-compose run --rm frontend bundle exec rspec
     docker-compose run --rm frontend bundle exec rake dev:lint
+    docker-compose -f docker-compose.yml -f docker-compose.minitest.yml run --rm minitest bundle exec rake test
```

Comment:
This guide is for people who learn contributing, not for documenting how to run old cruft. That guide is in the wiki

https://github.com/openSUSE/open-build-service/wiki/Development-Environment-Tips-&-Tricks#run-a-single-test-file-in-the-old-api-test-suite

Please remove this.

Reply:
ok, I'll update that wiki once it's merged

Reply:
removed

Reply:
circleci CLI doesnt work out of the box at least. Even without workflow, it *should* be possible to run a ```circleci build minitest``` but it doesn't. Maybe in a near future..